plugins/autoloader/prism-autoloader.js

```diff
@@ -253,42 +253,55 @@
 	}
 
 	/**
-	 * Returns the path to a grammar, using the language_path and use_minified config keys.
+	 * Returns all additional dependencies of the given element defined by the `data-dependencies` attribute.
 	 *
-	 * @param {string} lang
-	 * @returns {string}
+	 * @param {Element} element
+	 * @returns {string[]}
 	 */
-	function getLanguagePath(lang) {
-		return config.languages_path +
-			'prism-' + lang
-			+ (config.use_minified ? '.min' : '') + '.js'
+	function getDependencies(element) {
+		var deps = (element.getAttribute('data-dependencies') || '').trim();
+		if (!deps) {
+			var parent = element.parentElement;
+			if (parent && parent.tagName.toLowerCase() === 'pre') {
+				deps = (parent.getAttribute('data-dependencies') || '').trim();
+			}
+		}
+		return deps ? deps.split(/\s*,\s*/g) : [];
 	}
 
 	/**
-	 * Tries to load the grammar(s) and once loaded, highlights the given element again.
+	 * Returns whether the given language is currently loaded.
 	 *
 	 * @param {string} lang
-	 * @param {HTMLElement} elt
+	 * @returns {boolean}
 	 */
-	function registerElement(lang, elt) {
-		if (lang in lang_aliases) {
-			lang = lang_aliases[lang];
+	function isLoaded(lang) {
+		if (lang.indexOf('!') >= 0) {
```

Comment:
We seem to be doing some `force` checks with a variable in `load`. Is this code path actually hit or is this a few feature or what's the story here?

Reply:
Nevermind, I needed to reread the whole file.